doc/book/src/user/metrics.md

```diff
@@ -68,33 +68,39 @@ local instances of `prometheus` and `grafana` and provides a dashboard that
 charts several of the various metrics exposed by `zcashd`'s `prometheus`
 endpoint.
 
-`sudo docker-compose up` will start local instances of `prometheus` and `grafana`,
-accessible on ports `9090` and `3030` respectively. 
+`$(which docker-compose) up`[^1] will start local instances of `prometheus` and `grafana`,
```

Comment:
I don’t think the `which` is helpful outside of `sudo` (and, honestly, even with `sudo` I’d only use it in a script, for clarity.